doc/source/GettingStarted/TutorialAssemblingTheGafferBot/index.md

```diff
@@ -416,10 +416,10 @@ As Gaffy is looking a bit bland, you should drive the shader with some robot tex
 
 1. Create an Appleseed Color Texture node (_Appleseed_ > _Shader_ > _Texture2d_ > *asTexture*).
 
-2. In the _Node Editor_, point the node to the textures:
+2. In the Node Editor, point the node to the textures:
     - For the Filename plug, type `${GAFFER_ROOT}/resources/gafferBot/textures/base_COL/base_COL_<UDIM>.tx`.
 
-3. In the _Graph Editor_, connect the Appleseed color texture node's OutputColor plug to the Disney Material node's SurfaceColor plug. Gaffy's textures will now drive the color of the surface shader, and the render will update to show the combined results.
+3. In the Graph Editor, connect the Appleseed color texture node's OutputColor plug to the Disney Material node's SurfaceColor plug. Gaffy's textures will now drive the color of the surface shader, and the render will update to show the combined results.
```

Comment:
`Output Color` has a space in the UI 
![image](https://user-images.githubusercontent.com/896779/72746605-68fb4c00-3baa-11ea-9a94-56802dbd973a.png)